doc/classes/NavigationRegion2D.xml

```diff
@@ -1,13 +1,13 @@
 <?xml version="1.0" encoding="UTF-8" ?>
 <class name="NavigationRegion2D" inherits="Node2D" is_experimental="true" version="4.1" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="../class.xsd">
 	<brief_description>
-		A region of the 2D navigation map.
+		A traversable 2D region that [NavigationAgent2D]s can use for pathfinding.
```

Comment:
A region is not necessarily traversable, it can be disabled, locked or obstructed. The navigation mesh the region holds defines what is traversable while the navigation layers define what is accessible for a specific agent.